src/app/tests/TestEventLoggingNoUTCTime.cpp

```diff
@@ -80,57 +80,56 @@ class MockClock : public chip::System::Clock::ClockBase
     chip::System::Clock::ClockBase & mRealClock;
 };
 
-class TestContext : public chip::Test::AppContext
+class TestContext : public chip::Test::AppTestFixture
 {
 public:
     // Performs shared setup for all tests in the test suite
-    CHIP_ERROR SetUpTestSuite() override
+    static void SetUpTestSuite()
```

Comment:
make this call a static method if that is what is needed. Converting a subset of methods to be static is odd ... there is no clear reason of why some are static and some are not.

Maybe we should break down this PR into smaller chunks to define a good pattern on splitting things.

Reply:
I see googletest style wants these static. Marking resolved.